lib/bartleby/parser.ex

```diff
@@ -129,17 +121,15 @@ defmodule Bartleby.Parser do
     {:ok, nodes}
   end
 
-  defp _is_modult_tag_node?({:@, _l, [{:moduletag, _l2, [_text]}]}) do
-    true
-  end
+  defp _is_modult_tag_node?({:@, _l, [{:moduletag, _l2, [_text]}]}), do: true
   defp _is_modult_tag_node?(_), do: false
 
-  defp _is_doc_node?({:@, _l, [{:moduledoc, _l2, [_text]}]}) do
-    true
-  end
+  defp _is_doc_node?({:@, _l, [{:doc, _l2, [_text]}]}), do: true
+  defp _is_doc_node?({:@, _l, [{:moduledoc, _l2, [_text]}]}), do: true
   defp _is_doc_node?(_), do: false
 
   defp _parse_doc_node(node) when is_nil(node), do: []
+  defp _parse_doc_node(node) when is_list(node), do: Enum.map(node, &_parse_doc_node/1)
```

Comment:
Not the happiest that this is listed singular but it is plural and a list but couldnt think of something better.